plugins/discovery-azure-classic/src/test/java/org/elasticsearch/cloud/azure/classic/AbstractAzureComputeServiceTestCase.java

```diff
@@ -49,6 +66,10 @@ protected Settings nodeSettings(int nodeOrdinal) {
             .put(super.nodeSettings(nodeOrdinal))
             .put("discovery.zen.hosts_provider", "azure");
 
+        // Make the test run faster
+        builder.put(ZenDiscovery.JOIN_TIMEOUT_SETTING.getKey(), "50ms")
```

Comment:
I suspect these will be too small and we'll have time outs.

Reply:
That was the previous value in AzureMinimumMasterNodesTests and I moved it here so that other tests execute faster. Do you have any recommendation or should we use the default value?

Reply:
that's hard to say. 10ms ping cycle is way short (we use to make sure we do at least one but we simplified the code at the expense of losing this). How about 1s and see?